src/Create.js

```diff
@@ -0,0 +1,65 @@
+import { Component } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import Modal from "react-bootstrap/Modal";
```

Comment:
Great work on this team! 💯 

My only feedback here is:

Create a `BookFormModal` component (see task card 1 at the bottom). This separate component will contain the form elements required 👍